### PR TITLE
Linux

### PR DESCRIPTION
**Pull request by ocefpaf**
@ivoflipse let's see how far this takes us.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-      conda config --add channels conda-forge
-      conda config --set show_channel_urls true
-      conda install --yes --quiet conda-forge-build-setup
-      source run_conda_forge_build_setup
-script:
-  - conda build ./recipe
-
-after_success:
-
-  - ./ci_support/upload_or_check_non_existence.py ./recipe conda-forge --channel=main
```
Comment (ocefpaf) [inline .travis.yml]: We are not building on `OS X` so the `rerender` removed this correctly.

File: ci_support/run_docker_build.sh [hunk 2]
```diff
@@ -34,50 +38,18 @@ echo "$config" > ~/.condarc
...
-    export CONDA_PY=35
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    /feedstock_root/ci_support/upload_or_check_non_existence.py /recipe_root conda-forge --channel=main || exit 1
 
-    set -x
-    export CONDA_NPY=111
-    export CONDA_PY=35
-    set +x
+# Embarking on 1 case(s).
```
Comment (ocefpaf) [inline ci_support/run_docker_build.sh]: We do not need a `numpy` and `python` build matrix for `miktex`. This was also fixed by the rerender.

File: recipe/build.sh [hunk 3]
```diff
@@ -1,11 +1,4 @@
 #!/bin/bash
-# FIXME: This is a hack to make sure the environment is activated.
-# The reason this is required is due to the conda-build issue
-# mentioned below.
-#
-# https://github.com/conda/conda-build/issues/910
-#
-source activate "${CONDA_DEFAULT_ENV}"
```
Comment (ocefpaf) [inline recipe/build.sh]: I think that this was fixed in `conda-build`.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {% set version = "2.9.5987" %}
+{% set nix_version = "2.9-2016-08-17" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We need to think a little bit more about the version here :confused:

File: recipe/meta.yaml [hunk 5]
```diff
@@ -8,40 +9,40 @@ source:
...
-    - qt  # [linux]
-    - curl  # [linux]
-    - expat  # [linux]
-    - bzp2 1.0.*  # [linux]
-    - zlib 1.2.*  # [linux]
-    - libpng >=1.6.21,<1.7  # [linux]
+    - cmake  # [not win]
+    - bison  # [not win]
+    - flex  # [not win]
+    - qt  # [not win]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I had to add `bison` and `flex` to get the compilation going locally on the docker image but `cmake` is demanding `qt5` and we do not have that.

File: recipe/yum_requirements.txt [hunk 6]
```diff
@@ -1 +1,4 @@
-liburiparser-devel
```
Comment (ocefpaf) [inline recipe/yum_requirements.txt]: Maybe this is still need but I did not need it so far. (But I did get stuck at the `Qt5` dependency.)

File: recipe/yum_requirements.txt [hunk 7]
```diff
@@ -1 +1,4 @@
...
+ImageMagick
+dos2unix
+fop
+libxslt
```
Comment (ocefpaf) [inline recipe/yum_requirements.txt]: I am a little bit worried that I needed to add all those to get the compilation going...

**Comment by Korijn** [comment]
Maybe we can skip building `mpm_qt`?

http://miktex.org/howto/build-unx

We're fine with just the command line package manager.

**Comment by ocefpaf** [comment]
> Maybe we can skip building `mpm_qt`?

:+1:

**Comment by Korijn** [comment]
https://github.com/MiKTeX/miktex/blob/master/CMakeLists.txt#L296

Looks like there's an option to disable Qt.

File: recipe/meta.yaml (hunk 4 above)
Comment (ivoflipse) [inline recipe/meta.yaml]: Depending on how long compilation is going to take, I'd prefer if the Windows version also build from source, if only to keep all this straight

**Comment by ivoflipse** [comment]
Sorry I was looking on the feedstock for where this PR had come in and couldn't find it. Didn't realize you submitted it to our own fork >.< Anyway, thanks for looking into it!

**Comment by ocefpaf** [comment]
> Sorry I was looking on the feedstock for where this PR had come in and couldn't find it. 

No problem. I tried to make it easy for you and ended up making it more difficult :flushed:

**Pull request merged**